### ISSUE TITLE
PDF Full-text extraction does not seem to be indexed automatically in Solr in Hyrax 2.1.0

### ISSUE THREAD

**Issue by boveus** [samvera/hyrax#2967]
### Descriptive summary

We are using Hyrax version 2.1.0 

We enable the following flag in config/initializers/hyrax.rb to enable PDF full-text extraction
```ruby
  # Option to enable/disable full text extraction from PDFs
  # Default is true, set to false to disable full text extraction
  config.extract_full_text = true
```

We uploaded a pdf an example found [here](http://www.gulf-times.com/PDF) and attach it to a generic work type.  When we search for a keyword that is contained in the PDF in the 'all fields' search field from the homepage the PDF is not returned in the results.

There is a google group [thread](https://groups.google.com/forum/#!topic/samvera-tech/ogBCi1Gf6B8) regarding this issue, where we discovered a way to manually reindex Solr using the ActiveFedora::Base.reindex_everything method.  After manually reindexing Solr, we were able to run the same search and successfully return the PDF we were searching for.

We tested this on an OSX machine and Vagrant images in the development environment of Hyrax using rake hydra:server to launch and test our server.

### Rationale

It seems cumbersome to have to reindex Solr every time you add a new PDF.  

### Expected behavior

I would expect that the extracted full text of the PDF is indexed after the file upload.

### Actual behavior

The extracted full text of PDFs does not seem to be indexed automatically.

### Steps to reproduce the behavior

1. Set the config.extract_full_text value to 'true' on line 95 of config/initializers/hyrax.rb
2. Create a new work and attach a PDF file with text to the work.
3. Search for 'all fields' for a term that is contained within the PDF
4. Verify that there are no results
5. Open a rails console and run the ActiveFedora::Base.reindex_everything method.
6. Search for 'all fields' for a term that is contained within the PDF
7. Verify that there are results.

### Related work
There is a recent issue related to PDF full text extraction [here](https://github.com/samvera/hyrax/issues/2920)
This isn't related to this particular issue, but it involves the same functionality. 

### Collaborators
@sampofilms Nick Peterson 
@taliamath Talia Mathews 
@cgillen Corey Gillen 
@mjgiarlo Mike Giarlo

**Comment by no-reply** [comment]
This issue breaks full text indexing, which is turned on by default in Hyrax.

**Comment by no-reply** [comment]
@mjgiarlo is currently investigating.

**Comment by boveus** [comment]
It seems like this issue is resolved in the latest version of Hyrax 2.1.0.  Thank y'all!
